crates/net/network/src/test_only.rs

```diff
@@ -0,0 +1,74 @@
+
+#[cfg(test)]
+mod tests {
+    use crate::test_utils::Testnet;
+    use crate::transactions::TransactionPropagationMode::Max;
+    use crate::transactions::TransactionsManagerConfig;
+    use alloy_primitives::U256;
+    use rand::thread_rng;
+    use reth_provider::test_utils::ExtendedAccount;
+    use reth_provider::test_utils::MockEthProvider;
+    use reth_tracing::init_test_tracing;
+    use reth_transaction_pool::test_utils::TransactionGenerator;
+    use reth_transaction_pool::PoolTransaction;
+    use reth_transaction_pool::TransactionPool;
+    use tokio::time::Duration;
+
+    #[tokio::test]
+    async fn test_playground() {
```

Comment:
I'd actually like to keep this but ignore before we merge, I have a feeling this will be useful again